Psyche/Assets/Scripts/UI/OutroController.cs

```diff
@@ -10,83 +10,95 @@
 /// Author: jmolive8, blopezro, dnguye99
 public class OutroController : MonoBehaviour {
 
-    public GameObject outro1, outro2, outro3, outro4, clickText;
-    public Button quitButton, link;
-    private Image img1, img2, img3, img4, buttonImg, linkImg;
-    private TMP_Text buttonText;
+    [Header("Images")]
+    public GameObject Outro1;
+    public GameObject Outro2;
+    public GameObject Outro3;
+    public GameObject Outro4;
+    public GameObject ClickToContText;
+
+
+    [Header("Buttons")]
+    public Button QuitButton;
+    public Button LinkButton;
+
+    //variables
+    private Image Img1, Img2, Img3, Img4, ButtonImg, LinkImg;
+    private TMP_Text ButtonText;
     private bool IsTransitioning;
-    private Color buttonColor;
-    int curScreen = 1;
+    private Color ButtonColor;
+    int CurScreen = 1;

```

Comment:
I think this should have _private_ on the front.

Reply:
Corrected in commit [here](https://github.com/MissionToPsyche/tungsten_12a_web_game-se/pull/253/commits/9e0100f194fd60ed48e6f56ae8c83340000e2dde)